usaspending_api/reporting/tests/integration/test_agency_code_discrepancies.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+from model_mommy import mommy
+from rest_framework import status
+
+
+url = "/api/v2/reporting/agencies/123/discrepancies/?fiscal_year=2020&fiscal_period=2"
+
+
+@pytest.fixture
+def setup_test_data(db):
+    """ Insert test data into DB """
+    mommy.make(
+        "reporting.ReportingAgencyMissingTas",
+        toptier_code=123,
+        fiscal_year=2020,
+        fiscal_period=2,
+        tas_rendering_label="TAS 1",
+        obligated_amount=10.0,
+    )
+    mommy.make(
+        "reporting.ReportingAgencyMissingTas",
+        toptier_code=123,
+        fiscal_year=2020,
+        fiscal_period=2,
+        tas_rendering_label="TAS 2",
+        obligated_amount=1.0,
+    )
+    mommy.make(
+        "reporting.ReportingAgencyMissingTas",
+        toptier_code=321,
+        fiscal_year=2020,
+        fiscal_period=2,
+        tas_rendering_label="TAS 2",
+        obligated_amount=12.0,
+    )
+
```

Comment:
Might be worthwhile to create one more record which is never returned in the tests below.

Reply:
That one (obligated_amount=12.0) never is.

Reply:
Oh, thanks. different agency code